sbi/simulators/linear_gaussian.py

```diff
@@ -61,11 +61,11 @@ def get_true_posterior_log_prob_linear_gaussian_mvn_prior(
 
 
 def get_true_posterior_samples_linear_gaussian_uniform_prior(
-    observation: torch.Tensor, prior: Independent, num_samples: int = 1000, std=1,
+    x_o: torch.Tensor, prior: Independent, num_samples: int = 1000, std=1,
 ):
-    observation = utils.torchutils.atleast_2d(observation)
-    assert observation.ndim == 2, "needs batch dimension in observation"
-    mean = observation
+    x_o = utils.torchutils.atleast_2d(x_o)
+    assert x_o.ndim == 2, "needs batch dimension in observation"
```

Comment:
Full sentences for the user (capital... full stop.) I'd also favor the language 'Expected ...'